api/app/Models/AwardExperience.php

```diff
@@ -15,12 +14,12 @@
  * @property int $user_id
  * @property string $title
  * @property string $issued_by
- * @property Illuminate\Support\Carbon $awarded_date
+ * @property ?\Illuminate\Support\Carbon $awarded_date
  * @property string $awarded_to
  * @property string $awarded_scope
  * @property string $details
- * @property Illuminate\Support\Carbon $created_at
- * @property Illuminate\Support\Carbon $updated_at
+ * @property \Illuminate\Support\Carbon $created_at
+ * @property \Illuminate\Support\Carbon $updated_at
```

Comment:
Are we gonna use leading slash for all properties ? 
There are still left a lot this for $created_at  and probably for $updated_at too 
![Screenshot 2024-10-10 at 1 19 18 PM](https://github.com/user-attachments/assets/2061a7b7-cff0-4882-b87b-58fed87bc0ba)